docs/examples/simulation.md

```diff
@@ -0,0 +1,392 @@
+
+    To ensure security, avoid storing sensitive information like tokens directly in your code.
+    Instead, use environment variables.
+    Storing tokens in code shared on platforms like GitHub can lead to security incidents.
+    Anyone that possesses your token can impersonate you on the [CRIPT](https://criptapp.org/) platform.
+    Consider [alternative methods for loading tokens with the CRIPT API Client]().
+    In case your token is exposed be sure to immediately generate a new token to revoke the access of the old one
+    and keep the new token safe.
+
+```python
+import cript
+
+with cript.API(host="http://development.api.mycriptapp.org/", token="123456"):
+    pass
+```
+
+You may notice, that we are not executing any code inside the context manager block.
+If you were to write a python script, compared to a jupyter notebook, you would add all the following code inside that block.
+Here in a jupyter notebook, we need to connect manually. We just have to remember to disconnect at the end.
+
+```python
+api = cript.API("http://development.api.mycriptapp.org/", None)
+api = api.connect()
+```
+
+## Create a Project
+
+All data uploaded to CRIPT must be associated with a [project](../../nodes/primary_nodes/project) node.
+[Project](../../nodes/primary_nodes/project) can be thought of as an overarching research goal.
+For example, finding a replacement for an existing material from a sustainable feedstock.
+
+```python
+# create a new project in the CRIPT database
+project = cript.Project(name="My simulation project.")
+```
+
+## Create a Collection node
+
+For this project, you can create multiple collections, which represent a set of experiments.
+For example, you can create a collection for a specific manuscript,
+or you can create a collection for initial screening of candidates and one for later refinements etc.
+
+So, let's create a collection node and add it to the project.
+
+```python
+collection = cript.Collection(name="Initial simulation screening")
+# We add this collection to the project as a list.
+project.collection += [collection]
+```
+
+!!! note "Viewing CRIPT JSON"
+
+    Note, that if you are interested into the inner workings of CRIPT,
+    you can obtain a JSON representation of your data graph at any time to see what is being sent to the API.
+
+```python
+print(project.json)
+print("\nOr more pretty\n")
+print(project.get_json(indent=2).json)
+```
+
+## Create an Experiment node
+
+The [collection node](../../nodes/primary_nodes/collection) holds a series of
+[Experiment nodes](../../nodes/primary_nodes/experiment) nodes.
+
+And we can add this experiment to the collection of the project.
+
+```python
+experiment = cript.Experiment(name="Simulation for the first candidate")
+collection.experiment += [experiment]
+```
+
+# Create relevant Software nodes
+
+<a href="../../nodes/software" target="_blank">`Software`</a> nodes refer to software that you use during your simulation experiment. 
```

Comment:
how come we are not using a regular md link and instead opening the link in a new tab?

I'd keep everything in MD to make life simpler, but not a huge deal though